pipeline/transformer/transformer_sierra/src/test/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraAlternativeTitlesTest.scala

```diff
@@ -79,6 +79,20 @@ class SierraAlternativeTitlesTest
     getAlternativeTitles(varFields) shouldBe List("A", "B")
   }
 
+  it("should not include a subfield tagged 5 with content UkLW") {
+    val varFields = List(
+      createVarFieldWith(
+        "246",
+        "1",
+        List(
+          MarcSubfield(tag = "a", content = "A"),
+          MarcSubfield(tag = "5", content = "UkLW")
```

Comment:
Is it worth including two MARC subfields that:

* Have tag $5 but different content
* Have content UkLW but a different tag

and checking they're included?